src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -450,33 +438,32 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Bon voyage, aircraft!',
-            text: ['When the aircraft crosses the airspace boundary, it will ',
+            text: ['When the aircraft leaves your airspace , it will switch to center and',
                    'automatically remove itself from the flight strip bay on the right.',
-                   'Congratulations, you\'ve successfully taken off one aircraft.'
+                   'Congratulations, you\'ve successfully departed an aircraft.'
             ].join(' '),
             side: 'left',
             position: tutorial_position
         });
 
         this.tutorial_step({
             title: 'Arrivals',
-            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after',
-                   'you\'ve selected it, use the altitude/heading/speed controls you\'ve learned in',
-                   'order to guide it to be in front of a runway. Make sure to get the aircraft down to',
-                   'around 4,000ft, and 10-15 nautical miles (2-3 range rings) away from the airport.',
-                   'While you work the airplane, read the next step.'
+            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after you\'ve',
+                   'selected it, use the altitude/heading/speed controls you\'ve learned in order to',
+                   'guide it to the intercept of the ILS for the runway. The aircraft should be at an',
+                   'appropriate altitude and flying an appropriate heading before you clear it for approach!'
             ].join(' '),
             side: 'left',
             position: tutorial_position
         });
 
         this.tutorial_step({
             title: 'Approach Clearances, part 1',
-            text: ['You can clear aircraft for an ILS approach with the "ILS" command, followed by a runway name.',
-                   'When you do so, the aircraft will try to find a route that intercepts the localiser, represented by the',
-                   'extended centerline. Try giving radar vectors to get the aircraft on shallow intercept with this marking',
-                   'and then issue the instruction "i {RUNWAY}" to clear it to land. It should then guide itself',
-                   'the rest of the way to the runway, managing its own height and direction.'
+            text: ['You can clear aircraft for an ILS approach with the "i" command, followed by a runway name.',
+                   'When you do so, the aircraft will attempt to intercept the localiser, represented by the',
+                   'extended centerline. Try giving radar vectors to get the aircraft on an intercept of 30 degrees or less,',
+                   'and then issue the instruction "i {RUNWAY}" to clear it to land. It should then guide itself down',
```

Comment:
Strictly speaking, we're not clearing them to land

Reply:
Yeah sorry, that was actually from the old tutorial that I overlooked, I'll get that done.